packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -7,7 +7,7 @@ rememberDevice.timebased = 》Ðö ñöţ çĥåļļéñĝé ɱé öñ ţĥîš
 rememberDevice.devicebased = 》Ðö ñöţ çĥåļļéñĝé ɱé öñ ţĥîš ðéṽîçé åĝåîñ €홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 autoPush = 》Šéñð þûšĥ åûţöɱåţîçåļļý 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 unlockaccount = 》Ûñļöçķ åççöûñţ¿ ฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-haveaccount = 》Åļŕéåðý ĥåṽé åñ åççöûñţ ¿ 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+haveaccount = 》Åļŕéåðý ĥåṽé åñ åççöûñţ¿ 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
Thanks for making this change!

Am curious, did the error message provide sufficient instructions to resolve this?

Reply:
I believe so. (maybe part of reason is I'm aware the PL changes) :D